mixins/AuthenticationMixin.ts

```diff
@@ -49,9 +50,9 @@ export default class AuthenticationMixin extends Vue {
     }
   }
 
-  async SignUp(email: string, password: string, confirm_password: string) {
+  async SignUp(email: string, name: string, password: string, confirm_password: string) {
```

Comment:
Name this consistently with the google auth protocol -> the parameter should be called displayName